src/cobol/koopa/cobol/parser/preprocessing/replacing/ReplacingPhrase.java

```diff
@@ -64,6 +64,7 @@ public abstract boolean appliedTo(Source source,
 			Stack<Token> seen) {
 
 		List<Token> textWord = null;
+		boolean waitForClosingColon = false;
```

Comment:
Could we give this a more semantic name ? E.g. `buildingDummyOperand`, or something similar ?